### ISSUE TITLE
32Bit Version crashes when counter alias is requested

### ISSUE THREAD

**Issue by s3ta** [mickem/nscp#248]
We have 0.4.4.19 64Bit installed on a lot of machines and have no probs w/ our config.
Now we have some 32Bit-machines left on which we also wanted to use 0.4.4.19 Now we detected a lot of crashes running the service.
I tracked the problem down to this:
In the config we (re-)define counter for convenience 
```
[/settings/system/windows/counters]
avg_processes=\Objekte\Prozesse

```
Meanwhile I tried as well creating a paragraph like (yielding the same results as below)
```
[/settings/system/windows/counters/avg_processes]
counter=\Objekte\Prozesse
```

We have an alias command for this:
`alias_procs= Check_pdh "warn=value> 100" "crit=value > 200" counter=avg_processes`

If I run `nscp test` and do `alias_procs` I get `Failed to process command: bad allocation`
as well if I run `Check_pdh counter=avg_processes` manually.

If I run `Check_pdh counter=\\Objekte\Prozesse` I get a reasonable/the desired result.

(no similar problem w/ 64Bit - there I can run any of the commands above
Problem occurs w/ 0.4.4.15 as well) 

Any Ideas?
Can/Should I provide dump files?
Can I check anything in the dump files?

**Comment by mickem** [comment]
Are you sure it is related to aliases and not just some counters?

**Comment by mickem** [comment]
Try this build and let me know if the issue is resolved: https://github.com/mickem/nscp/releases/tag/0.5.0.24

**Comment by s3ta** [comment]
Thanks - I'll try the version if we'll hit the next 32Bit-System while migrating.
(I don't want to touch the meanwhile running system w/ Version 0.4.1.105) 

And yes, I'm pretty sure it is related to aliases, since I can query the counter directly - see above.

**Comment by mickem** [comment]
You can always grab the zip and run it stand alone if you do not want to install it...

**Comment by mickem** [comment]
This fix is now in the latest nightly build of 0.4.4 as well.

**Comment by mickem** [comment]
Closing this, please reopen if this is still an issue.
